### PR TITLE
[trivial][but contentious] Make ModelLoader naming consistent

### PR DESCRIPTION
**Pull request by bertmaher**
*Description*: OK, so I get that C++ isn't Java... but these classes have a non-trivial hierarchy, and when I started working on it, I kept getting confused by trivial things (like, is ModelLoader a base class?  Which file defines it?).  I'm sure some of this is a personal problem, but I also feel like it doesn't hurt to make things a bit more consistent in this area.  What do you all think?
*Testing*: `ninja check`
*Documentation*: N/A

### CODE REVIEW

**Comment by jfix71** [comment]
I'm a big fan but I hesitate to approve until others have weighed in 😄

File: include/glow/Importer/ONNXIFIModelLoader.h [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
-#include "glow/Importer/ONNX.h"
+#include "glow/Importer/ONNXModelLoader.h"
 
 #include "llvm/ADT/StringMap.h"
 
 namespace glow {
 namespace onnxifi {
 
-class ModelLoader : public ONNXModelLoader {
+class ONNXIFIModelLoader : public ONNXModelLoader {
```
Comment (rdzhabarov) [inline include/glow/Importer/ONNXIFIModelLoader.h]: it's already in ```onnxifi``` namespace. I think it makes sense to keep a short name for the class.

**Comment by bertmaher** [comment]
That sounds good to me.  I noticed the `onnxifi` namespace, but I'd still rather have a descriptive class name, since `ModelLoader` sounds like a base class.   (Sorry for the bikeshedding 🚲🏘️  :-D ).

File: include/glow/Importer/ONNXIFIModelLoader.h [hunk 2]
```diff
@@ -19,16 +19,15 @@
 
 #include "onnx/onnxifi.h"
 
-#include "glow/Importer/ONNX.h"
+#include "glow/Importer/ONNXModelLoader.h"
```
Comment (rdzhabarov) [inline include/glow/Importer/ONNXIFIModelLoader.h]: can you fix include guards as well

**Pull request merged**